grails-app/conf/UrlMappings.groovy

```diff
@@ -4,6 +4,13 @@ class UrlMappings {
 
         "/patents" controller: "patent", action: [POST: "patents"]
 
+        "/$regionType/$regionName/patents" (controller: 'speciesIp', action: 'region') {
```

Comment:
Why do we need need this webservice? I thought there would be a batch job that automatically fetches the data from auspat. How I am supposed to use this when deployed?